### PR TITLE
Update PowerBI API Rest client library with new report operations

### PR DESCRIPTION
**Pull request by ArinaHantsis**

### CODE REVIEW

File: sdk/PowerBI.Api/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -24,5 +24,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
```
Comment (ranbreuer) [inline sdk/PowerBI.Api/Properties/AssemblyInfo.cs]: no need to update assembly here, Build system takes care of it

Comment (ArinaHantsis) [inline sdk/PowerBI.Api/Properties/AssemblyInfo.cs]: Thanks.

File: sdk/swagger.json [hunk 2]
```diff
@@ -834,6 +834,116 @@
...
+            "name": "requestParameters",
+            "description": "The entity to post",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CloneReportRequest"
+            }
+          }
+        ],
+        "responses": {
```
Comment (ranbreuer) [inline sdk/swagger.json]: no other relevant status codes?

File: sdk/swagger.json [hunk 3]
```diff
@@ -834,6 +834,116 @@
...
+          {
+            "name": "reportKey",
+            "description": "The report id",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "requestParameters",
+            "description": "The entity to post",
```
Comment (ali-hamud) [inline sdk/swagger.json]: in place of
The entity to post

consider:
clone report parameters

File: sdk/swagger.json (hunk 2 above)
Comment (ali-hamud) [inline sdk/swagger.json]: we should fix this in all APIs.
later.

File: sdk/swagger.json [hunk 4]
```diff
@@ -1196,22 +1306,50 @@
...
-    },
-    "CreateWorkspaceRequest": {
-      "type": "object",
-      "description": "A Create Workspace Request",
-      "properties": {
-        "name": {
-          "type": "string",
-          "description": "The workspace name"
-        }
-      }
```
Comment (ArinaHantsis) [inline sdk/swagger.json]: Why did you remove it?
